components/components-googledrive/components-googledrive-integration/src/test/java/org/talend/components/google/drive/runtime/GoogleDriveBaseTestIT.java

```
@@ -0,0 +1,189 @@
+
+    protected String testTS;
+
+    protected GoogleDriveSource source;
+
+    protected GoogleDriveSink sink;
+
+    protected GoogleDriveConnectionProperties connectionProperties;
+
+    protected GoogleDriveCreateProperties createProperties;
+
+    protected GoogleDriveCreateRuntime createRuntime;
+
+    protected GoogleDriveListProperties listDriveProperties;
+
+    protected RuntimeContainer container = new RuntimeContainer() {
+
+        private Map<String, Object> map = new HashMap<>();
+
+        @Override
+        public Object getComponentData(String componentId, String key) {
+            return map.get(componentId + "_" + key);
+        }
+
+        @Override
+        public void setComponentData(String componentId, String key, Object data) {
+            map.put(componentId + "_" + key, data);
+        }
+
+        @Override
+        public String getCurrentComponentId() {
+            return "GoogleDrive-ITs";
+        }
+
+        @Override
+        public Object getGlobalData(String key) {
+            return null;
+        }
+
+    };
+
+    protected transient static final Logger LOG = getLogger(GoogleDriveBaseTestIT.class);
+
+    static {
+        GOOGLEDRIVE_SERVICE_ACCOUNT_FILE = System.getProperty(GOOGLEDRIVE_SERVICE_ACCOUNT_PROP);
+    }
+
+    /**
+     * Sometimes, we may have some SocketTimeOut exceptions, in that case we don't consider that the test has failed. It
+     * won't break the whole build.
+     */
+    @Rule
+    public TestRule skipRule = new TestRule() {
+
+        public Statement apply(final Statement base, Description desc) {
+            return new Statement() {
+
+                public void evaluate() throws Throwable {
+                    try {
+                        base.evaluate();
+                    } catch (SocketTimeoutException ex) {
+                        // If called with an expression evaluating to false, the test will halt and be ignored.
+                        Assume.assumeTrue(false);
```

This is useless in this scope as the exception is already catched and ignored.

I don't know why you need to ignore this error but please at least add a log of it.

---

I agree that I should log it but this exception is not ignored and catch during tests.

---

Sorry I wasn't clear enough. I'm talking about the assume method call. It's useless as the exception is already catched and ignored.
So I think that you can remove this call